application/view/omeka/migrate/index.phtml

```diff
@@ -1,9 +1,13 @@
-<?php
-$translate = $this->plugin('translate');
-?>
-<h2><?php echo $translate('Update database'); ?></h2>
-<p><?php echo $translate('There was a recent software upgrade and the database needs to be updated. The
-entire site will be down for maintenance until you click the button below.'); ?></p>
+<h2><?php echo $this->translate('Update database'); ?></h2>
+<p><?php echo $this->translate('There was a recent software upgrade and the database needs to be updated. The entire site will be down for maintenance until you click the button below.'); ?></p>
 <form method="post">
-    <button name="submit"><?php echo $translate('Update database'); ?></button>
+    <button name="submit"<?php echo $environment->isCompatible() ? '' : ' disabled'; ?>><?php echo $this->translate('Update database'); ?></button>
 </form>
+<?php if (!$environment->isCompatible()): ?>
+<p><?php echo $this->translate('There are environment errors that must be resolved before you can update the database.'); ?></p>
```

Comment:
There should be some sort of handle here for styling purposes, a div with a class wrapping around both the paragraph and the list probably?

@kimisgold could have input

Reply:
My main suggestion would be that the list be set to `<ul class="messages">` and each error use the `<li class="error">` markup.